src/classes/genre.rb

```diff
@@ -0,0 +1,15 @@
+class Genre
+  attr_accessor :name
+  attr_reader :id, :items
+
+  def initialize(name, items, id = rand(0..500))
```

Comment:
class Genre should have only property shown in the diagram what is the name. As genre has many items kindly change @items to empty array